Find a functionally equivalent code:android.media.MediaFocusControl.discardAudioFocusOwner:COMMENT
Method Modifier: protected   
Comment:/**
 * Discard the current audio focus owner.
 * Notify top of audio focus stack that it lost focus (regardless of possibility to reassign
 * focus), remove it from the stack, and clear the remote control display.
 */

@@ -5,14 +5,10 @@
             final FocusRequester exFocusOwner = mFocusStack.pop();
             exFocusOwner.handleFocusLoss(AudioManager.AUDIOFOCUS_LOSS);
             exFocusOwner.release();
-            // clear RCD
-            synchronized (mRCStack) {
-                clearRemoteControlDisplay_syncAfRcs();
-            }
         }
     }
 }

